packages/ui/src/composites/wui-input-text/styles.ts

```diff
@@ -102,6 +101,17 @@ export default css`
     left: var(--wui-spacing-l);
   }
 
+  .wui-size-mdl {
+    padding: 17.25px var(--wui-spacing-m) 17.25px var(--wui-spacing-m);
+  }
+  wui-icon + .wui-size-mdl,
+  wui-loading-spinner + .wui-size-mdl {
+    padding: 17.25px var(--wui-spacing-3xl) 17.25px 40px;
```

Comment:
If possible, let's use variable for `40px` as well. (Prob we don't have it but not sure)